marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/steps/CustomStepController.java

```diff
@@ -33,11 +37,33 @@
         return ResponseEntity.ok(newService().getCustomStep(stepName));
     }
 
+    @RequestMapping(method = RequestMethod.POST)
+    @ApiImplicitParam(required = true, paramType = "body", dataType = "StepSchema")
+    @Secured("ROLE_writeCustom")
+    public ResponseEntity<Void> createCustomStep(@RequestBody @ApiParam(hidden = true) ObjectNode propertiesToAssign) {
```

Comment:
There is no need for this endpoint. I believe users will be able to only edit custom steps from HC. They cannot create one

Reply:
Fixed